### DISCUSSION TITLE
What would be our toolset?

### DISCUSSION THREAD

**Discussion by felipeschneider88** [dataplat/DataSaturdays#126] [Ideas]
Based on the reply from @way0utwest on # 81 and #117, I'm starting this thread to talk about our toolset. I think we need to be the owners of the entire stack of the toolset. That being said, we need to start to define the tools we need to support the org.

A general idea for starting the debate is:
* A paying gateway 
     * An users (for events, payments and auth)
     * A form submissions 
     * some API-like backend to integrate with 3rd party payment systems (Paypal, credit card, etc)
* A CFS, Agenda, we can use https://sessionize.com/ for free like now
* A presentation platform (MS Teams, zoom, Google meets, etc)
* The website 
     * An `[semi]` auto-approval process for new events
     * Some CI/CD pipeline to build newer versions
     * Accessibility, I know this is a whole topic but want to get lost in the seas of words and discussions :) 
   
As I said before, this is a work in progress list and nothing on the above list isn't written in stone. Everyone is welcome, so feel free to add/remove

**Comment by way0utwest** [comment]
I might clarify (and add)

- API integration for CFP systems (i.e. sessionize, callfordataspeakers, etc.)
- API integration for ticketing (i.e. Eventbrite)
- tooling for admins to customize their site (date, verbiage, logo, etc.)
- reporting for sponsors/vendors
- API integration for presentation platforms (zoom, hangouts, YT, Streamyard, Teams, etc.)
- CoC/AHP agreement and customization for events

Other than that, I like the list, and I do think we want to start building out tools here. 

There also needs to be a separate discussion for architecture, based on volunteers' skills and willingness.
